codecov.yml

```diff
@@ -0,0 +1,35 @@
+# This config overrides the default Codecov config, as outlined at:
+# https://docs.codecov.io/docs/codecov-yaml
+
+coverage:
+  round: up
+  range: "70...100"
+  precision: 2
+
+  status:
+    project:
+      default:
+        target: auto
+        threshold: "0%"
```

Comment:
We might want to tweak these targets & thresholds (see also below for the `patch` key) but these seem sensible for now.